molgenis-core-ui/src/main/javascript/modules/react-components/AlertMessage.js

```diff
@@ -1,14 +1,14 @@
-/* global _: false, React: false, molgenis: true */
-(function(_, React, molgenis) {
-	"use strict";
-	
+import React from "react";
+import DeepPureRenderMixin from "./mixin/DeepPureRenderMixin";
+import Icon from "./Icon";
+
 	var div = React.DOM.div, span = React.DOM.span, button = React.DOM.button;
```

Comment:
What is the style guide on indents?
I would expect this not indented (many more places)